wrangler-transform/src/e2e-test/resources/testData/Wrangler/parse_HL7_Wrangler-cdap-data-pipeline (1).json

```diff
@@ -3,7 +3,7 @@
     "description": "Data Pipeline Application",
     "artifact": {
         "name": "cdap-data-pipeline",
-        "version": "6.10.0-SNAPSHOT",
+        "version": "[6.0.0, 7.0.0]",
```

Comment:
nit: `[6.0.0, 7.0.0)`

this comment applies to whole PR.

Reply:
changed in all the json pipelines.

Reply:
This comment is not addressed still, the upper bound should not be included `[6.0.0, 7.0.0)`

Reply:
changed.